administrator/components/com_admin/sql/updates/postgresql/5.1.0-2024-02-10.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "#__guidedtour_steps" ADD COLUMN "params" text NOT NULL /** CAN FAIL **/;
```

Comment:
```suggestion
ALTER TABLE "#__guidedtour_steps" ADD COLUMN "params" text NULL /** CAN FAIL **/;
```
See previous review comment.